gwpy/timeseries/timeseries.py

```diff
@@ -1805,6 +1805,57 @@ def whiten(self, fftlength=None, overlap=0, method=DEFAULT_FFT_METHOD,
         out = in_.convolve(tdw, window=window)
         return out * numpy.sqrt(2 * in_.dt.decompose().value)
 
+    def find_gates(self, tzero=1.0, whiten=True,
+                   threshold=50., cluster_window=0.5, **whiten_kwargs):
+        """Identify points that should be gates using a provided threshold
+        and clustered within a provided time window.
+
+        Parameters
+        ----------
+        tzero : `int`, optional
+            half-width time duration (seconds) in which the timeseries is
+            set to zero
+
+        whiten : `bool`, optional
+            if True, data will be whitened before gating points are discovered,
+            use of this option is highly recommended
+
+        threshold : `float`, optional
+            amplitude threshold, if the data exceeds this value a gating window
+            will be placed
+
+        cluster_window : `float`, optional
+            time duration (seconds) over which gating points will be clustered
+
+        **whiten_kwargs
+            other keyword arguments that will be passed to the
+            `TimeSeries.whiten` method if it is being used when discovering
+            gating points
+
+        Returns
+        -------
+        out : `~gwpy.segments.SegmentList`
+            a list of segments that should be gated based on the
+            provided parameters
+        See also
```

Comment:
Need a blank line here

```suggestion

        See also
```